src/QA/QA.Web/Mappings/ReviewMapper.cs

```diff
@@ -125,6 +121,8 @@ public async Task<ReviewViewModel> Map(VacancyReview review)
         {
             var vacancy = review.VacancySnapshot;
 
+            var updatedVacancy = _vacancyClient.GetVacancyAsync(review.VacancyReference);
```

Comment:
You'll need to add this to the WhenAll. Maybe change this to `currentVacancy` as it may not have been updated

Reply:
i have done it.